HISTORY.rst

```diff
@@ -13,6 +13,7 @@ Announcements
 New features and enhancements
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 * Added the ability to search for simulations that reach a given warming level. (:pull:`251`).
+* New argument attribute_weights to ``generate_weights`` to allow for custom weights. (:pull:`252`).
 
 Breaking changes
```

Comment:
You'll have to add the `experiment_weights` renaming to the breaking changes below.